packages/nuxt-locokit/src/runtime/styles/primevue/_datepicker.scss

```diff
@@ -35,21 +36,21 @@
   border: 0 none;
   background: transparent;
   border-radius: 50%;
-  transition: color 0.15s, box-shadow 0.15s;
+  transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
 }
 
 .p-datepicker .p-datepicker-header .p-datepicker-prev:enabled:hover,
 .p-datepicker .p-datepicker-header .p-datepicker-next:enabled:hover {
   color: var(--text-color);
   border-color: transparent;
-  background: transparent;
+  background: #f3f4f6;
```

Comment:
Same here, I'm not convinced to set hard-coded color values in those files...

Reply:
It is legacy from v0.7 or improvement to adapt new style
See https://github.com/locokit/locokit/issues/171